From a4b397f11a03c55b066c6fcfb9a0bff8dc6f10fc Mon Sep 17 00:00:00 2001
From: Jack Huang <jackhuang021@gmail.com>
Date: Thu, 11 Jul 2024 23:50:06 +0800
Subject: [PATCH] drivers: nvme: add NVME_QUIRK_NO_NS_DESC_LIST for sm2263xtv

Signed-off-by: Jack Huang <jackhuang021@gmail.com>
---
 drivers/nvme/host/pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/nvme/host/pci.c b/drivers/nvme/host/pci.c
index 886c3fc9578e..4d06296d3e97 100644
--- a/drivers/nvme/host/pci.c
+++ b/drivers/nvme/host/pci.c
@@ -3460,6 +3460,8 @@ static const struct pci_device_id nvme_id_table[] = {
 	{ PCI_DEVICE(0x126f, 0x2263),	/* Silicon Motion unidentified */
 		.driver_data = NVME_QUIRK_NO_NS_DESC_LIST |
 				NVME_QUIRK_BOGUS_NID, },
+	{ PCI_DEVICE(0x1e95, 0x9100),	/* Silicon Motion sm2263xtv */
+		.driver_data = NVME_QUIRK_NO_NS_DESC_LIST },
 	{ PCI_DEVICE(0x1bb1, 0x0100),   /* Seagate Nytro Flash Storage */
 		.driver_data = NVME_QUIRK_DELAY_BEFORE_CHK_RDY |
 				NVME_QUIRK_NO_NS_DESC_LIST, },
-- 
2.34.1

